index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <main>
+        <header>
+            <nav>
+                <a href=""><img src="./assets/images/Logo Placeholder.png" alt=""></a>
+                <a href="http://"><img src="./assets/images/Enabled.png" alt="my logo"></a>
+            </nav>
+        </header>
+
+        <section>
+            <h1>
+                I'm Abdy.<br /> Glad to see <br /> you!
+            </h1>
+            <div class="content">
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+                    work and experience! If you like what you see and have a project you need coded, don’t hesitate to
+                    contact me.</p>
+            </div>
+            <h2 class="connect-link">LET'S CONNECT</h2>
+            <div class="social-media-icons">
+                <ul class="social-media-links">
+                    <li>
+                        <a href="http://">
+                            <img src="./assets/images/twiter.png" alt="twiter">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="http://">
+                            <img src="./assets/images/Linkedin icon.png" alt="linkedin">
```

Comment:
- [ ] You have not build the require project according to the requirementOn tjis milestone  you are require to build the middle section of the mobile website. We included a screenshot of Template 1 below for reference, but you should stick to the template that you chose. This sections of the mobile website should include:presently you only build the 

"Works" section
<img width="188" alt="about-myself-mobile (2)" src="https://github.com/kakalygaday/Portfolio-Project/assets/92458236/6999d121-7e2f-49ca-bf89-dcaf85303e26">
- About section 
- 
![works-mobile (2)](https://github.com/kakalygaday/Portfolio-Project/assets/92458236/2dd1efff-0be0-487b-b6f4-50a938c79a95)


your project 

https://github.com/kakalygaday/Portfolio-Project/assets/92458236/d9eeda0c-a20e-43c6-8144-2e89bdc9c7bf